src/BrowserPane.h

```diff
@@ -38,21 +38,18 @@
 @end
 
 @interface BrowserPane : NSView<BaseView, WebUIDelegate, WebFrameLoadDelegate, MMTabBarItem> {
-	TabbedWebView * webPane;
 	IBOutlet NSButton * backButton;
 	IBOutlet NSButton * forwardButton;
 	IBOutlet NSButton * refreshButton;
 	IBOutlet NSButton * rssPageButton;
 	IBOutlet SSTextField * addressField;
 	IBOutlet NSButton * iconImage;
 	IBOutlet NSImageView * lockIconImage;
-	IBOutlet NSView * viewwwer;
 	NSString * pageFilename;
 	NSError * lastError;
 	BOOL isLocalFile;
 	BOOL isLoading;
 	BOOL hasRSSlink;
-	NSString * viewTitle;
```

Comment:
instead of declaring it here, synthesizing it in the .m and having to write a standard getter and setter, we should just declare it as `@property` directly

Reply:
You are right, seems like I had a temporary red/green blindness, and confused - with + at the same time 🤦‍♂️ The main point of the review was the other one anyway.